Niamh25/doc 5675/vault named query syntax (#1617)
corda/corda-docs-portal: content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/vault-queries.md

```
{{% /tab %}}
{{< /tabs >}}

# Vault-Named Query Operators

> The following list describes the operators and right operand types for the vault-named query syntax:

Reviewer: ```suggestion
The following is the list of the operators and right operand types for the vault-named query syntax:
```
The verb "describes" has an incorrect meaning in this sentence.

> 

Reviewer 2: Do these match the subsections below?

Reviewer 2: Oh, ok. I think that could be clearer. Should that bulleted list not include ALL of the operators, including those explained further below (perhaps with links)? Also, is "and right operand types" correct?

* `IN`
* `LIKE`
* `IS NOT NULL`
* `IS NULL`
* `AS`
* `OR`
* `AND`
* `!=`
* `>`
* `<`
* `>=`
* `<=`

> ### Detailed Operators

Reviewer 2: ```suggestion
### Detailed Operators
```
Is this necessary?

Author: I added the second header to distinguish that only some operators had descriptions and not the entire list. Do you think it would be better to not have it?

Reviewer 2: This relates to my other comment. I think if you had a full list, with links where applicable, this heading isn't necessary.

Could add maybe add a sentence such as the following:

Where the behaviour is not standard, the operators are explained in detail in the following sections..

> *Name:* `->`

Reviewer: ```suggestion
**Name:** `->`
```
Sorry, my bad, you need ** to make the text appear bold. Please update.

> <b> Name: </b> `->`

Reviewer: ```suggestion
*Name:* `->`
```
Change all instances of the bold tag to * to follow our markdown convention.
